charts/jenkins/README.md

```diff
@@ -1,11 +1,14 @@
-# Jenkins
+# jenkins
 
+![Version: 5.0.18](https://img.shields.io/badge/Version-5.0.18-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 2.440.1](https://img.shields.io/badge/AppVersion-2.440.1-informational?style=flat-square)
```

Comment:
I would prefer not to have these badges, it generate unneeded churn in the readme, even if values are not changing it means this needs to change

Reply:
If both the (major) changes to this file are not really desired/needed I'll remove the README.md.gotmpl and revert this file change. These badges would be auto-changed however by helm-docs. Let me know if you indeed want this file to be reverted.

Reply:
I think we can revert it.